### PR TITLE
add-ng-delegate-relnotes-784xx

### PR DESCRIPTION
**Pull request by kat-enos**
Adds delegate release notes for the NextGen 784xx release slated for 2/13 or thereabouts.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by xingchi-jin — APPROVED** [review]
LG for me
@AnupamIO   pl. also double check the helm part

File: release-notes/delegate.md [hunk 1]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+Harness NextGen release 784xx includes the following changes for Harness Delegate.
+
+:::note
+The repository location of the Helm chart for the NextGen delegate is changing. (DEL-5576)
+
+The repository is being deprecated. Updates to the chart will not be made to https://app.harness.io/storage/harness-download/delegate-helm-chart/ and will not be available from that location. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
+:::
+
+### New features and enhancements
```
Comment (KimberlyFields) [inline release-notes/delegate.md:29]: Revise the headings throughout to match the headings of the other release notes: What's new, Early access, Fixed issues.

File: release-notes/delegate.md [hunk 2]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
 
+## February 13, 2023, version 784xx
+### Delegate version 78306
+
+Harness NextGen release 784xx includes the following changes for Harness Delegate.
+
+:::note
+The repository location of the Helm chart for the NextGen delegate is changing. (DEL-5576)
+
+The repository is being deprecated. Updates to the chart will not be made to https://app.harness.io/storage/harness-download/delegate-helm-chart/ and will not be available from that location. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
```
Comment (KimberlyFields) [inline release-notes/delegate.md:26]:
```suggestion
The repository at https://app.harness.io/storage/harness-download/delegate-helm-chart/ is being deprecated. The Helm chart in https://app.harness.io/storage/harness-download/delegate-helm-chart/ will no longer be updated and will no longer be available from that location. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
```

File: release-notes/delegate.md [hunk 3]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+:::
+
+### New features and enhancements
+
+This release introduces the following new features and enhancements:
+
+- Added the `helm repo update` command to the delegate installer. The command is included in the instructions that apply the delegate manifest. This change reduces the chance of retrieving the wrong file from the repository. (DEL-5540)
```
Comment (KimberlyFields) [inline release-notes/delegate.md:33]: Also add this to the overall What's new topic.

Comment (pratmit) [inline release-notes/delegate.md:33]: This release introduces command line friendly wizard for delegate installation using Helm Chart, Terraform, Kubernetes manifest or docker.

Comment (pratmit) [inline release-notes/delegate.md:33]: This release introduces a way to copy the delegate token from the delegate tokens page in app.harness.io

File: release-notes/delegate.md [hunk 4]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes). For Self-Managed Enterprise Edition, see [Self-Managed Enterprise Edition release notes](/release-notes/self-managed-enterprise-edition).
 
 
+## February 13, 2023, version 784xx
```
Comment (kat-enos) [inline release-notes/delegate.md:18]:
```suggestion
## February 14, 2023, version 78421
```

File: release-notes/delegate.md [hunk 5]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+### Delegate version 78306
+
+Harness NextGen release 784xx includes the following changes for Harness Delegate.
```
Comment (kat-enos) [inline release-notes/delegate.md:21]:
```suggestion
Harness NextGen release 78421 includes the following changes for Harness Delegate.
```

File: release-notes/delegate.md [hunk 6]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+:::note
+The repository location of the Helm chart for the NextGen delegate is changing. (DEL-5576)
+
+The repository at https://app.harness.io/storage/harness-download/delegate-helm-chart/ is being deprecated. The Helm chart in https://app.harness.io/storage/harness-download/delegate-helm-chart/ will no longer be updated and will no longer be available from that location. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
```
Comment (kat-enos) [inline release-notes/delegate.md:26]:
```suggestion
The repository at https://app.harness.io/storage/harness-download/delegate-helm-chart/ is being deprecated. The Helm chart will no longer be available from the repository at   https://app.harness.io/storage/harness-download/delegate-helm-chart/. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
```

File: release-notes/delegate.md [hunk 7]
```diff
@@ -15,6 +15,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+:::
+
+### New features and enhancements
```
Comment (kat-enos) [inline release-notes/delegate.md:29]:
```suggestion
### What's new
```

File: release-notes/delegate.md (hunk 3 above)
Comment (kat-enos) [reply release-notes/delegate.md:33]: @pratmit , I need the ticket for these additions or an indication of where the content goes. I assume that the above and below items would go in the **What's new** section. Correct?

File: release-notes/delegate.md (hunk 1 above)
Comment (kat-enos) [reply release-notes/delegate.md:29]: Did in the file.

File: release-notes/delegate.md (hunk 3 above)
Comment (deepakbolangady) [reply release-notes/delegate.md:33]: I did not see this item in the overall what's new file so I copied it over before publishing. @kat-enos, please review and make any corrections that might be necessary. I scanned the what's new file for earlier instances of delegate sections to see whether the title would be "### Delegate version <version>" but I did not see anything I could reproduce. Thanks.

**Review by deepakbolangady — APPROVED** [review]
Approving after updating the date (title and metadata) and copying the what's new item from the delegate release notes to the overall what's new release notes file. Katherine, please review anyway. Thanks.

**Review by deepakbolangady — APPROVED** [review]

File: release-notes/delegate.md (hunk 3 above)
Comment (pratmit) [reply release-notes/delegate.md:33]: This should go in the What's new section.

**Pull request merged**